os/arch/arm/Kconfig

```diff
@@ -266,6 +267,8 @@ config ARCH_CORTEXA32
         default n
         select ARCH_DCACHE
         select ARCH_ICACHE
+	select ARCH_HAVE_IRQPRIO
+	select ARCH_HAVE_RESET
```

Comment:
It's minor, please align the indentation.

Reply:
Good catch, thankyou